shared/haml/user_header.haml

```diff
@@ -2,23 +2,33 @@
 :ruby
   require 'cdo/create_header'
   require 'cdo/language_dir'
-  limit_project_types = request.cookies["_limit_project_types_#{rack_env}"]
 
-  level = defined?(local_assigns) ?
-    local_assigns[:level] :
-    false
+  # Determine if we want to show the create menu
+  if local_assigns[:show_create_menu]
```

Comment:
Do you need to check `defined?` here? (`defined?(local_assigns)`)

Reply:
My understanding is that `local_assigns` is a method (https://apidock.com/rails/v5.0.0.1/ActionView/Template/local_assigns) so we don't need to check if it's `defined?`  (I think the original code wasn't checking what it looked like it was checking for.)